src/app/containers/SendInvoke/index.js

```diff
@@ -27,13 +29,15 @@ class SendInvoke extends Component {
     this.setState({
       [key]: e.target.value,
     })
+    logDeep('key: ', key + ' ' + e.target.value)
   }
 
   _handleArgChange = (id, e) => {
-    const myArgs = this.state.args
+    var myArgs = this.state.args
```

Comment:
Why the need for var here? (and other places)

Reply:
This is one that slipped by.